src/java.se/share/data/jdwp/jdwp.spec

```diff
@@ -2149,10 +2149,10 @@ JDWP "Java(tm) Debug Wire Protocol"
             (Error THREAD_NOT_SUSPENDED)
             (Error OPAQUE_FRAME      "Attempted to return early from "
                                      "a frame corresponding to a native "
-                                     "method. "
-                                     "The thread is a virtual thread and the target "
-                                     "VM is unable force its current frame to return. "
-                                     "Or the implementation is "
+                                     "method, "
+                                     "or the thread is a virtual thread and the target "
+                                     "VM is unable to force its current frame to return, "
+                                     "or the implementation is "
```

Comment:
It's okay to join the sentences but if you do then we need to re-forward L2152 so that we don't have a line break after the comma at L2152.

Reply:
I don't understand. Aren't they all formatted into one long line, and line wrapping is controlled by how the browser formats the table cell?

https://docs.oracle.com/en/java/javase/17/docs/specs/jdwp/jdwp-protocol.html#JDWP_ThreadReference_ForceEarlyReturn

Reply:
Yes, that's the generated spec. The comment was on the source file which can be cleaned up, or later if you want, it doesn't matter.

Reply:
Ok, so you just want the spec to be better formatted. I can fix that now.